apps/studio/components/interfaces/Connect/DatabaseConnectionString.tsx

```diff
@@ -135,7 +135,10 @@ export const DatabaseConnectionString = () => {
       ? getConnectionStrings({
```

Comment:
Should this fix also be applied in `apps/studio/components/interfaces/Connect/ConnectTabContent.tsx`?

Reply:
oh yes! thank you so much 🙏